test/Autofac.Extensions.DependencyInjection.Test/Autofac.Extensions.DependencyInjection.Test.csproj

```diff
@@ -42,8 +44,10 @@
     <PackageReference Include="Microsoft.Extensions.Options" Version="3.0.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp3.1'">
```

Comment:
Why the additional condition block? We should really only test against a single version of the Microsoft DI packages.

Reply:
Actually in this case I think it does make sense to have several frameworks tested.
So far behavior of `Microsoft.Extensions.DependencyInjection.Specification.Tests` has not changed, but it might?!

I can remove it if you want, would definitely make things easier.

Reply:
I don't expect any major behaviour to the DI specification right now; possibly one to review after .NET 5.